test/Microsoft.AspNet.Routing.Tests/Template/TemplateParserTests.cs

```diff
@@ -634,10 +688,7 @@ public void InvalidTemplate_InvalidParameterNameWithOpenBracketThrows()
         {
             ExceptionAssert.Throws<ArgumentException>(
                 () => TemplateParser.Parse("{a}/{a{aa}/{z}"),
```

Comment:
Do the same for default values, as it was not possible to put a reserved character in a default before.